code/_onclick/hud/screen_objects/base_screen_objects.dm

```diff
@@ -443,6 +458,8 @@
 //--------------------------------------------------nsa---------------------------------------------------------
 /obj/screen/nsa
 	name = "nsa"
+	desc = "Neural System Accumulation is a variable, that shows how many augmentations, mutations and chemicals you can handle.\
```

Comment:
```suggestion
	desc = "Neural System Accumulation depicts strain your body is experiencing.\
	<br>It is increased by chemicals and mutations.\
```
Since we have NSA and NSA limit, you explained NSA limit and HUD tells more about former than the latter